content/community/tools-starter-kits.md

```diff
@@ -5,34 +5,37 @@ layout: community
 permalink: community/starter-kits.html
 ---
 
-## Recommended by the React Team {#recommended-by-the-react-team}
+## Recomendados pelo time do React {#recommended-by-the-react-team}
 
-* **[Create React App](https://github.com/facebook/create-react-app)** - An officially supported way to start a client-side React project with no configuration
-* **[Next.js](https://nextjs.org/)** - Framework for server-rendered or statically-exported React apps
-* **[Gatsby](https://www.gatsbyjs.org/)** - Static site generator for React optimized for speed
-* **[nwb](https://github.com/insin/nwb)** - A toolkit for React apps, libraries and other npm modules for the web
-* **[razzle](https://github.com/jaredpalmer/razzle)** - Create server-rendered universal JavaScript applications with no configuration
-* **[Neutrino](https://neutrino.js.org/)** - Create and build modern JavaScript applications with zero initial configuration
-* **[Parcel](https://parceljs.org)** - Fast, zero configuration web application bundler
+* **[Create React App](https://github.com/facebook/create-react-app)** - Uma maneira oficial de iniciar um projeto React do lado do cliente sem configuração.
+* **[Next.js](https://nextjs.org/)** - Framework para aplicativos React renderizados por servidor ou exportados estaticamente.
+* **[Gatsby](https://www.gatsbyjs.org/)** - Gerador de site estático para React otimizado para velocidade.
+* **[nwb](https://github.com/insin/nwb)** - Um kit de ferramentas para aplicativos React, bibliotecas e outros módulos npm para a web.
+* **[razzle](https://github.com/jaredpalmer/razzle)** - Criar aplicativos JavaScript universais renderizados pelo servidor sem configuração.
+* **[Neutrino](https://neutrino.js.org/)** - Crie e construa aplicativos JavaScript modernos com configuração inicial zero.
+* **[Parcel](https://parceljs.org)** - Rápido, bundler de aplicativo web de configuração zero.
 
-## Other Starter Kits {#other-starter-kits}
+## Outros kits para iniciantes {#other-starter-kits}
```

Comment:
```suggestion
## Outros Kits Iniciais {#other-starter-kits}
```
Não é necessariamente um kit para "iniciantes" e sim um kit "para início". Falar que é para iniciante dá uma impressão que é um kit para ajudar os inexperientes, quando na verdade é um kit de eficiência, para iniciar mais rápido "direto ao ponto".